easylist/easylist_specific_hide.txt

```diff
@@ -842,6 +842,7 @@ experienceproject.com##.adn
 chat-avenue.com##.adnoyer_log
 flightglobal.com##.adp
 vice.com##.adph
+apkdone.com##.adplaceholder-top
```

Comment:
Any particular reason you didn't suggest this as a generic hide?

Reply:
It's shown on mobile too and uBO disables generic cosmetic by default on mobile. I'll switch to generic if requested, but then I'll add the specific filter to uAssets too.